src/app.js

```diff
@@ -0,0 +1,50 @@
+const auth = require('./auth');
+const error = require('koa-error');
+const Koa = require('koa');
+const morgan = require('koa-morgan');
+const Router = require('koa-router');
+const session = require('koa-session');
+const ts = require('./tsapi');
+const views = require('koa-views');
+
+const app = new Koa();
+const router = new Router();
+
+// nicer error pages for development
+app.use(error());
+
+// create session - needed by passport for authentication
+// random key used so oauth dance is performed every time
+// TODO: implement refresh token strategy and use a constant key
```

Comment:
leaving TODO comments in; is the implementation we have here good enough for now?

Reply:
yeah, it needs to be